paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -104,21 +107,43 @@ class PaparazziPlugin : Plugin<Project> {
         val nonTransitiveRClassEnabled =
           (project.findProperty("android.nonTransitiveRClass") as? String).toBoolean()
 
+        // local resources
+        val localResourceFiles = project
+          .files(variant.sourceSets.flatMap { it.resDirectories })
+          .asFileTree
+
+        // library resources
+        // https://android.googlesource.com/platform/tools/base/+/96015063acd3455a76cdf1cc71b23b0828c0907f/build-system/gradle-core/src/main/java/com/android/build/gradle/tasks/MergeResources.kt#875
+        val libraryResourceFiles = variant.runtimeConfiguration
+          .incoming
+          .artifactView { config: ArtifactView.ViewConfiguration ->
+            config.attributes { container: AttributeContainer ->
+              container.attribute(
+                AndroidArtifacts.ARTIFACT_TYPE,
+                AndroidArtifacts.ArtifactType.ANDROID_RES.type
+              )
+            }
+          }
+          .artifacts
+          .artifactFiles
+          .asFileTree
+
         task.packageName.set(android.packageName())
```

Comment:
@jrodbx Do we need this part behind `app.cash.paparazzi.new.resource.loading`? It does not harm to load these resources paths when using old `resourceResolver`

Reply:
nope!  should be fine to load both paths in the plugin